### PR TITLE
minor changes in javascript

### PR DESCRIPTION
**Pull request by oddasif**

### CODE REVIEW

File: Week2_Javascript/asif/index.html [hunk 1]
```diff
@@ -210,13 +210,15 @@ <h5>Points to remember</h5>
                     <div class="form-group col-1 w20">:</div>
                     <div class="form-group col-6">
                         <select id="head-of-ac" name="head-of-ac" class="form-control required">
-                            <option value="" disabled selected>Select</option>
+                            
+                            <option value="-" disabled selected>Select</option>
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/index.html:214]: Remove value "-" and disabled attribute

File: Week2_Javascript/asif/index.html [hunk 2]
```diff
@@ -241,11 +243,12 @@ <h5>Points to remember</h5>
                     <div class="form-group col-1 w20">:</div>
                     <div class="form-group col-6">
                         <select id="expenditure-type" name="expenditure-type" class="form-control">
-                            <option value="" disabled selected>Select</option>
+                            <option value="-" disabled selected>Select</option>
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/index.html:246]: Remove value "-" and disabled attribute

File: Week2_Javascript/asif/index.html [hunk 3]
```diff
@@ -254,10 +257,9 @@ <h5>Points to remember</h5>
                     <div class="form-group col-1 w20">:</div>
                     <div class="form-group col-6">
                         <select id="purpose-type" name="purpose-type" class="form-control">
-                            <option value="" disabled selected>Select</option>
-                            <!-- <option value="" id="purpose-type-val-1">Option 1</option>
-                            <option value="" id="purpose-type-val-2">Option 2</option> -->
+                            <option value="-" disabled selected>Select</option>
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/index.html:260]: Remove value "-" and disabled attribute

File: Week2_Javascript/asif/script.js [hunk 4]
```diff
@@ -101,35 +162,28 @@ function validateForm() {
...
-        }
-        // else if(containsSpecialChars(ifscCode.substr(5, 10))) {
-        //     setError("ifsc-code", "*Last six characters should be numeric or alphabets only");
-        //     returnval = false;
-        // }
 
+
+    // for expenditure type
+    var expType = document.getElementById("expenditure-type").value;
+    if(expType == "-") {
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/script.js:175]: Here check value is empty or null

File: Week2_Javascript/asif/script.js [hunk 5]
```diff
@@ -101,35 +162,28 @@ function validateForm() {
...
+        setError("expenditure-type", "*required");
+        returnval = false;
     }
 
 
+    // for purpose type
+    var purposeType = document.getElementById("purpose-type").value;
+    if(purposeType == "-") {
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/script.js:183]: Here check value is empty or null

File: Week2_Javascript/asif/script.js [hunk 6]
```diff
@@ -101,35 +162,28 @@ function validateForm() {
...
-    var ifscCode = document.getElementById("ifsc-code").value;
-    if(ifscCode.length == 0) {
-        setError("ifsc-code", "*required");
-        returnval = false;
-    }
-    else if(ifscCode.length != 11) {
-        setError("ifsc-code", "*Should be of 11 characters only");
+    // for hod
+    var hod = document.getElementById("head-of-ac").value;
+    if(hod == "-") {
```
Comment (prakashPixelvide) [inline Week2_Javascript/asif/script.js:167]: Here check value is empty or null

**Review by prakashPixelvide — COMMENTED** [review]
Some point commented fix that

**Comment by oddasif** [comment]
done with the changes

**Pull request merged**